migrator/migrations/frozenschema/v75/notification_schedules.go

```diff
@@ -0,0 +1,33 @@
+package schema
```

Comment:
I'm not sure what the new etiquette will be for the migrations, but the readme mentions post 3.74 that frozen schemas may be kept within the migration and not put into a shared place (currently two migrations follow this), I'm fine with it either way but just wanted to note it might be worthwhile being consistent.

Reply:
I made this migration consistent with the recent migrations.